api/IDBRequest.json

```
@@ -182,6 +182,51 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "transient_unrecoverable_read_errors": {
+            "__compat": {
+              "description": "Improved reporting for read errors",
+              "spec_url": "https://w3c.github.io/IndexedDB/#:~:text=NotReadableError",
+              "support": {
+                "chrome": [
+                  {
+                    "version_added": "132",
+                    "notes": "Returns a `NotReadableError` exception for unrecoverable read errors, and an `UnknownError` for transient read errors."
+                  },
+                  {
+                    "version_added": "130",
+                    "partial_implementation": true,
+                    "notes": "Returns a `NotFoundError` exception for unrecoverable read errors, and a `DataError` for transient read errors."
+                  }
+                ],
+                "chrome_android": "mirror",
+                "edge": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
```

See: https://github.com/mdn/browser-compat-data/pull/25277#discussion_r1883682922

```suggestion
                  "version_added": "7"
```